content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -793,6 +799,15 @@ This system is the account number of a financial account.
       "value": "5646"
     }
 
+##### Bill Codes Types
+
+This system is the bill code type for a charge item.`Bill code type` can be either `CDM_SCHED`, `CPT`, `HCPCS`,`ICD`, `MODIFIER` or `REVENUE`.
```

Comment:
nit: space after the first sentence.

Reply:
updated [here|https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R804]